src/main/java/uk/gov/justice/digital/delius/transformers/StaffTransformer.java

```diff
@@ -27,9 +28,21 @@ public static StaffDetails staffDetailsOf(Staff staff) {
                         .map(TeamTransformer::teamOf)
                         .collect(Collectors.toList()))
                 .probationArea(probationAreaOf(staff.getProbationArea(), false))
+                .staffGrade(new KeyValue(staff.getGrade() != null ? staff.getGrade().getCodeValue() : null,staff.getGrade() != null ? staff.getGrade().getCodeDescription() : null))
```

Comment:
You should be able to use `KeyValueTransformer.keyValueOf` here. It takes a StandardReference.

```suggestion
            .staffGrade(KeyValueTransformer.keyValueOf(staff.getGrade()))
```